stages/org.osbuild.greenboot

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python3
+"""
+Configure greenboot
+
+Update configuration of greenboot in /etc/greenboot/greenbot.conf.
+"""
+
+import fileinput
+import sys
+
+import osbuild.api
+
+
+SCHEMA = """
+"additionalProperties": false,
+"required": ["config"],
+"properties": {
+  "config": {
+    "additionalProperties": false,
+    "description": "greenboot config options",
+    "type": "object",
+    "properties": {
+      "monitor_services": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+      }
+    }
+  }
+}
+"""
+
+
+def main(tree, options):
+    greenboot_conf = options.get("config", {})
+    changes = greenboot_conf.copy()
+    config_file = f"{tree}/etc/greenboot/greenboot.conf"
+    key_to_conf = {
+        "GREENBOOT_MONITOR_SERVICES": "monitor_services"
+    }
+
+    # For each of the configured options, find the first non-commented out instance
+    # of the option and replace it (if necessary). If it does not already exist, append
+    # the option to the end of the file.
+    # Keys are case case sensitive.
+    with fileinput.input(files=(config_file), inplace=True) as f:
+        for line in f:
+            if line.startswith("#"):
+                sys.stdout.write(line)
+                continue
+            line_list = line.split('=')
+            if len(line_list) != 2:
+                sys.stdout.write(line)
+                continue
+            key, current = line_list
+            key_in_conf = key_to_conf.get(key, None)
+            if key_in_conf is None:
+                continue
```

Comment:
Noticed one more issue here (sorry, didn't catch it earlier).

If the key isn't known (it's not in the `key_to_conf`), we drop the line.  Should write it out as is before continuing.

Reply:
ah right, I've mainly took from https://github.com/osbuild/osbuild/blob/main/stages/org.osbuild.pwquality.conf so now the early continue(s) need a write to the file, fixed thanks